File: web-be/src/main/java/com/project/web_be/dtos/AnswerDTO.java
package com.project.web_be.dtos;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.project.web_be.entities.Answer;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class AnswerDTO {
    @JsonProperty("answer_text")
    private String answerText;
    @JsonProperty("is_correct")
    private boolean isCorrect;
    @JsonProperty("question_id")
    private Long questionId;

}

File: web-fe/src/app/dtos/requests/answer.dto.ts
export class AnswerDTO {
    answer_text: string;
    is_correct: boolean;
    question_id!: number;

    constructor(data: any) {
        this.answer_text = data.answer_text;
        this.is_correct = data.is_correct;
        this.question_id = data.question_id;
    }
}
